bgpd/bgp_route.c

```diff
@@ -6094,6 +6100,9 @@ static void route_vty_out_route(struct prefix *p, struct vty *vty,
 		prefix2str(p, buf, PREFIX_STRLEN);
 		len = vty_out(vty, "%s", buf);
 #endif
+	} else if (p->family == AF_FLOWSPEC) {
+		route_vty_out_flowspec(vty, p, NULL,
```

Comment:
Please,  provide an example of output either into the commit log and/or into the documentation.

Reply:
https://gist.github.com/pguibert6WIND/f23f34528f7c6f03c3e6832dcefcbece

Reply:
I hesitate with 3 solutions that re not yet pushed for bgp flowspec nice display.
Actually, flowspec entries can be huge when adding proto, port range,...:

bgpd# show bgp ipv4 flowspec
*>i proto = 6 , = 17  srcp > 1024  flags tcp flags is exactly match 32, or tcp flags is exactly match 4 pktlen > 200 , < 300 , > 400 , < 500                                100

solution 1 : do nothing
solution 2 : fit the entry description in the first 40 bytes. if the text goes over the 40 bytes, an extra line will be added and will also be limited to 40 bytes... until the end of the text
solution 3 : because the rule can contain or rules, display could be simplified by splitting the rules in as many places as there are combinatories.

Actually, solution 2 seems to me good approach. For solution 3, I am still thinking about it.

Reply:
thank you, I do not have good comments. It is a mess to be displayed.

Reply:
As a thought here, how does the underlying `ip XXXX` linux commands display this?